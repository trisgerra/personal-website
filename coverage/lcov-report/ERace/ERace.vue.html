<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for ERace/ERace.vue</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">ERace</a> ERace.vue
    </h1>
    <div class='clearfix'>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">20x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">&lt;template&gt;
  &lt;div class="erace columns"&gt;
    &lt;iframe class="frame-stream column is-10-desktop is-8-tablet is-12-mobile" src="https://player.twitch.tv/?channel=asraces" frameborder="0" allowfullscreen="no" scrolling="no"&gt;&lt;/iframe&gt;
    &lt;iframe class="frame-chat column is-2-desktop is-4-tablet is-hidden-mobile" src="https://www.twitch.tv/embed/asraces/chat?darkpopout" frameborder="0" scrolling="no"&gt;&lt;/iframe&gt;
  &lt;/div&gt;
&lt;/template&gt;
&nbsp;
&lt;script lang="ts"&gt;
import { Component, Vue } from 'vue-property-decorator';
&nbsp;
@Component
export default class ERace extends Vue {
&nbsp;
}
&lt;/script&gt;
&nbsp;
&lt;style lang="scss"&gt;
.erace {
  height: 96vh;
&nbsp;
  .column{
    padding: 0px;
  }
}
&nbsp;
iframe {
  height: inherit;
}
&lt;/style&gt;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Apr 14 2020 12:35:07 GMT+0200 (Central European Summer Time)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
