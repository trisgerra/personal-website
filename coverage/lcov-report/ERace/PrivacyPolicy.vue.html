<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for ERace/PrivacyPolicy.vue</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">ERace</a> PrivacyPolicy.vue
    </h1>
    <div class='clearfix'>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">&lt;template&gt;
  &lt;!-- eslint-disable max-len --&gt;
  &lt;div&gt;
  &lt;h2&gt;ASRaces Terms of Service&lt;/h2&gt;
  &lt;h3&gt;1. Terms&lt;/h3&gt;
  &lt;p&gt;By accessing the app ASRACES , you are agreeing to be bound by these terms of service, all applicable laws and regulations, and agree that you are responsible for compliance with any applicable local laws. If you do not agree with any of these terms, you are prohibited from using or accessing this site. The materials contained in this website are protected by applicable copyright and trademark law.&lt;/p&gt;
  &lt;h3&gt;2. Use License&lt;/h3&gt;
  &lt;ol type="a"&gt;
    &lt;li&gt;Permission is granted to temporarily download one copy of the materials (information or software) on ASRaces, non-commercial transitory viewing only. This is the grant of a license, not a transfer of title, and under this license you may not:
      &lt;ol type="i"&gt;
        &lt;li&gt;use the materials for any commercial purpose, or for any public display (commercial or non-commercial);&lt;/li&gt;
        &lt;li&gt;attempt to decompile or reverse engineer any Software included in ASRaces apps (iOS &amp; Android)&lt;/li&gt;
        &lt;li&gt;transfer the materials to another person or "mirror" the materials on any other server.&lt;/li&gt;
      &lt;/ol&gt;
    &lt;/li&gt;
    &lt;li&gt;This license shall automatically terminate if you violate any of these restrictions and may be terminated by ASRaces at any time. Upon terminating your viewing of these materials or upon the termination of this license, you must destroy any downloaded materials in your possession whether in electronic or printed format.&lt;/li&gt;
  &lt;/ol&gt;
  &lt;h3&gt;3. Disclaimer&lt;/h3&gt;
  &lt;ol type="a"&gt;
    &lt;li&gt;The materials on ASRaces iOS and Android are provided on an 'as is' basis. ASRaces makes no warranties, expressed or implied, and hereby disclaims and negates all other warranties including, without limitation, implied warranties or conditions of merchantability, fitness for a particular purpose, or non-infringement of intellectual property or other violation of rights.&lt;/li&gt;
    &lt;li&gt;Further, ASRaces does not warrant or make any representations concerning the accuracy, likely results, or reliability of the use of the materials on its website or otherwise relating to such materials or on any sites linked to this site.&lt;/li&gt;
  &lt;/ol&gt;
  &lt;h3&gt;4. Limitations&lt;/h3&gt;
  &lt;p&gt;In no event shall ASRaces or its suppliers be liable for any damages (including, without limitation, damages for loss of data or profit, or due to business interruption) arising out of the use or inability to use the materials on ASRaces's apps, even if ASRaces authorized representative has been notified orally or in writing of the possibility of such damage. Because some jurisdictions do not allow limitations on implied warranties, or limitations of liability for consequential or incidental damages, these limitations may not apply to you.&lt;/p&gt;
  &lt;h3&gt;5. Accuracy of materials&lt;/h3&gt;
  &lt;p&gt;The materials appearing on ASRaces's website could include technical, typographical, or photographic errors. ASRaces does not warrant that any of the materials on its website are accurate, complete or current. ASRaces may make changes to the materials contained on its website at any time without notice. However ASRaces does not make any commitment to update the materials.&lt;/p&gt;
  &lt;h3&gt;6. Links&lt;/h3&gt;
  &lt;p&gt;ASRaces has not reviewed all of the sites linked to its website and is not responsible for the contents of any such linked site. The inclusion of any link does not imply endorsement by ASRaces of the site. Use of any such linked website is at the user's own risk.&lt;/p&gt;
  &lt;h3&gt;7. Modifications&lt;/h3&gt;
  &lt;p&gt;ASRaces may revise these terms of service for its website at any time without notice. By using this website you are agreeing to be bound by the then current version of these terms of service.&lt;/p&gt;
  &lt;h3&gt;8. Governing Law&lt;/h3&gt;
  &lt;p&gt;These terms and conditions are governed by and construed in accordance with the laws of Italy and you irrevocably submit to the exclusive jurisdiction of the courts in that State or location.&lt;/p&gt;
  &lt;h3&gt;9. Data stored&lt;/h3&gt;
  &lt;p&gt;This application use Google Analytics to track the usage for each country in order to provide better features to the users.&lt;/p&gt;
&lt;/div&gt;
&lt;/template&gt;
&nbsp;
&lt;script lang="ts"&gt;
import { Component, Vue } from 'vue-property-decorator';
import MobileApp from '@/pages/ERace/components/MobileApp.vue';
import Twitch from './components/Twitch.vue';
import Races from './components/Races.vue';
&nbsp;
@Component({
  components: {
    MobileApp,
    Twitch,
    Races,
  },
})
export default class ERace extends Vue {
&nbsp;
}
&lt;/script&gt;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sun Jul 05 2020 22:18:32 GMT+0200 (Central European Summer Time)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
