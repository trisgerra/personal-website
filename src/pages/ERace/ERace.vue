<template>
  <div class="erace">
    <iframe class="frame-stream" src="https://player.twitch.tv/?channel=asraces" frameborder="0" allowfullscreen="no" scrolling="no" :height="$vssHeight" :width="$vssWidth-350"></iframe>
    <iframe cleass="frame-chat" src="https://www.twitch.tv/embed/asraces/chat" frameborder="0" scrolling="no" height="500" width="350"></iframe>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';
import VueScreenSize from 'vue-screen-size';

@Component({
  mixins: [VueScreenSize.VueScreenSizeMixin],
})
export default class ERace extends Vue {

}
</script>

<style lang="scss">
.erace {
  height: 100vh;
}

iframe {
  height: inherit;
}
</style>
