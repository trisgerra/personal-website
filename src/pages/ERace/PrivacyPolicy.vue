<template>
  <!-- eslint-disable max-len -->
  <div class="privacy">

  <h1><button class="button" @click="() => {
              this.$router.go(-1)
            }">Go Back</button>ASRaces Terms of Service</h1>
  <h3>1. Terms</h3>
  <p>By accessing the app ASRACES , you are agreeing to be bound by these terms of service, all applicable laws and regulations, and agree that you are responsible for compliance with any applicable local laws. If you do not agree with any of these terms, you are prohibited from using or accessing this site. The materials contained in this website are protected by applicable copyright and trademark law.</p>
  <h3>2. Use License</h3>
  <ol type="a">
    <li>Permission is granted to temporarily download one copy of the materials (information or software) on ASRaces, non-commercial transitory viewing only. This is the grant of a license, not a transfer of title, and under this license you may not:
      <ol type="i">
        <li>use the materials for any commercial purpose, or for any public display (commercial or non-commercial);</li>
        <li>attempt to decompile or reverse engineer any Software included in ASRaces apps (iOS & Android)</li>
        <li>transfer the materials to another person or "mirror" the materials on any other server.</li>
      </ol>
    </li>
    <li>This license shall automatically terminate if you violate any of these restrictions and may be terminated by ASRaces at any time. Upon terminating your viewing of these materials or upon the termination of this license, you must destroy any downloaded materials in your possession whether in electronic or printed format.</li>
  </ol>
  <h3>3. Disclaimer</h3>
  <ol type="a">
    <li>The materials on ASRaces iOS and Android are provided on an 'as is' basis. ASRaces makes no warranties, expressed or implied, and hereby disclaims and negates all other warranties including, without limitation, implied warranties or conditions of merchantability, fitness for a particular purpose, or non-infringement of intellectual property or other violation of rights.</li>
    <li>Further, ASRaces does not warrant or make any representations concerning the accuracy, likely results, or reliability of the use of the materials on its website or otherwise relating to such materials or on any sites linked to this site.</li>
  </ol>
  <h3>4. Limitations</h3>
  <p>In no event shall ASRaces or its suppliers be liable for any damages (including, without limitation, damages for loss of data or profit, or due to business interruption) arising out of the use or inability to use the materials on ASRaces's apps, even if ASRaces authorized representative has been notified orally or in writing of the possibility of such damage. Because some jurisdictions do not allow limitations on implied warranties, or limitations of liability for consequential or incidental damages, these limitations may not apply to you.</p>
  <h3>5. Accuracy of materials</h3>
  <p>The materials appearing on ASRaces's website could include technical, typographical, or photographic errors. ASRaces does not warrant that any of the materials on its website are accurate, complete or current. ASRaces may make changes to the materials contained on its website at any time without notice. However ASRaces does not make any commitment to update the materials.</p>
  <h3>6. Links</h3>
  <p>ASRaces has not reviewed all of the sites linked to its website and is not responsible for the contents of any such linked site. The inclusion of any link does not imply endorsement by ASRaces of the site. Use of any such linked website is at the user's own risk.</p>
  <h3>7. Modifications</h3>
  <p>ASRaces may revise these terms of service for its website at any time without notice. By using this website you are agreeing to be bound by the then current version of these terms of service.</p>
  <h3>8. Governing Law</h3>
  <p>These terms and conditions are governed by and construed in accordance with the laws of Italy and you irrevocably submit to the exclusive jurisdiction of the courts in that State or location.</p>
  <h3>9. Data stored</h3>
  <p>This application use Google Analytics to track the usage for each country in order to provide better features to the users.</p>
</div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

  @Component({
    components: {},
  })
export default class PrivacyPolicy extends Vue {
}
</script>

<style lang="scss">
  .privacy {
    margin: 30px;

    .button {
      margin-right: 15px
    }

    h1 {
      font-weight: bold;
      font-size: 25px;
    }

    h2 {
      font-weight: bold;
      font-size: 20px;
    }
    h3 {
      font-weight: bold;
      font-size: 16px;
    }
    p {
      font-size: 16px;
    }
  }
</style>
