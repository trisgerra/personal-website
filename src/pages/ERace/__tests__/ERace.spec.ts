import { Wrapper } from '@vue/test-utils';
import createWrapper from '@/utils/CreateWrapper';
import ERace from '../Erace.vue';

describe('ERace', () => {
  it('should render as expected', () => {
    const wrapper: Wrapper<ERace> = createWrapper(ERace);
    expect(wrapper.element).toMatchSnapshot();
  });
});
