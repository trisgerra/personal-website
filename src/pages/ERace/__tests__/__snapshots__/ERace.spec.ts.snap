// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ERace should render as expected 1`] = `
<div
  class="erace"
>
  <iframe
    allowfullscreen="allowfullscreen"
    class="frame-stream"
    frameborder="0"
    height="768"
    scrolling="no"
    src="https://player.twitch.tv/?channel=asraces"
    width="674"
  />
   
  <iframe
    cleass="frame-chat"
    frameborder="0"
    height="500"
    scrolling="no"
    src="https://www.twitch.tv/embed/asraces/chat"
    width="350"
  />
</div>
`;
